tests/specs/mcd/word-pack.k

```diff
@@ -22,50 +22,119 @@ endmodule
 module WORD-PACK-JAVA [kast]
     imports WORD-PACK-COMMON
 
-    rule    ADDR_UINT48_UINT48 ==K #WordPackAddrUInt48UInt48(ADDR, UINT48_1, UINT48_2)
-         => ADDR     ==Int maxUInt160 &Int  ADDR_UINT48_UINT48
-    andBool UINT48_1 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow160)
-    andBool UINT48_2 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow208)
-    andBool #rangeUInt(256, ADDR_UINT48_UINT48)
-      [simplification]
-
-    rule    UINT48_UINT48 ==K #WordPackUInt48UInt48(UINT48_1, UINT48_2)
-         => UINT48_1 ==Int maxUInt48 &Int  UINT48_UINT48
-    andBool UINT48_2 ==Int maxUInt48 &Int (UINT48_UINT48 /Int pow48)
-    andBool #rangeUInt(96, UINT48_UINT48)
-      [simplification]
-
-    rule    ADDR_UINT8 ==K #WordPackAddrUInt8(ADDR, UINT8)
-         => ADDR  ==Int maxUInt160 &Int  ADDR_UINT8
-    andBool UINT8 ==Int maxUInt8   &Int (ADDR_UINT8 /Int pow160)
-    andBool #rangeUInt(168, ADDR_UINT8)
-      [simplification]
+    //
+    // #WordPack decomposition
+    // 
+    rule       ADDR_UINT48_UINT48:Int ==Int #WordPackAddrUInt48UInt48(ADDR:Int, UINT48_1:Int, UINT48_2:Int) 
+            => ADDR     ==Int maxUInt160 &Int  ADDR_UINT48_UINT48              
+       andBool UINT48_1 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow160)
+       andBool UINT48_2 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow208)
+       andBool #rangeUInt(256, ADDR_UINT48_UINT48)
+       [simplification]
+
+    rule       #WordPackAddrUInt48UInt48(ADDR:Int, UINT48_1:Int, UINT48_2:Int) ==Int ADDR_UINT48_UINT48:Int
+            => ADDR     ==Int maxUInt160 &Int  ADDR_UINT48_UINT48              
+       andBool UINT48_1 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow160)
+       andBool UINT48_2 ==Int maxUInt48  &Int (ADDR_UINT48_UINT48 /Int pow208)
+       andBool #rangeUInt(256, ADDR_UINT48_UINT48)
+       [simplification]
```

Comment:
```suggestion
    rule       #WordPackAddrUInt48UInt48(ADDR:Int, UINT48_1:Int, UINT48_2:Int) ==Int ADDR_UINT48_UINT48:Int
            => ADDR_UINT48_UINT48:Int ==Int #WordPackAddrUInt48UInt48(ADDR:Int, UINT48_1:Int, UINT48_2:Int)
       [simplification]
```

To reduce change of error/typo. Similarly for the others.

Reply:
See: https://github.com/runtimeverification/k/issues/2587

Reply:
Yes, the `[comm]` attribute will be very helpful there.